Automattic/wp-calypso
# Site Profiler: Update migration CTAs to redirect to the full URL

Fixes https://github.com/Automattic/dotcom-forge/issues/7567

## Proposed Changes

* On the hosting section, update the CTA to use the final URL (with the protocol)
* On the migration banner down the page, redirect to the same link.

## Why are these changes being made?

To have a consistent behavior on all the migration links on the page

## Testing Instructions

* Go to `/site-profiler/:site` on a non-WP.com site. Ex: `/site-profiler/quintoandar.com.br`
* On the hosting section, click on the link below the title
* You should be redirected to the migration flow with the URL pre-filled in the new page (with the protocol)
* Go to the migration banner on the bottom of the page
* You should be redirected to the migration flow with the URL pre-filled in the new page (with the protocol)

| Hosting section  | Migration banner |
| ------------- | ------------- |
|![CleanShot 2024-06-03 at 16 09 58@2x](https://github.com/Automattic/wp-calypso/assets/5039531/1dfdd79a-c04f-462a-86c5-66a5fa3af8ed)|![CleanShot 2024-06-03 at 16 10 14@2x](https://github.com/Automattic/wp-calypso/assets/5039531/0051426c-a6b1-4b73-af84-f150a75e986c)








## Pre-merge Checklist
